drivers/frequency/ad9517/ad9517.c

```diff
@@ -115,7 +115,11 @@ int32_t ad9517_setup(struct ad9517_dev **device,
 	/* Update the device with user settings for the LVPECL output
 	 * channels. */
 	for(index = 0; index < 4; index++) {
-		reg_address = AD9517_REG_LVPECL_OUT0 + index;
+		if (index < 2)
```

Comment:
This code seems correct but if for me a bit cryptic to understand and probably will be cryptic for others reading it in the future. One other approach would be:
```
static uint16_t lvepcl_out_ch[4] = {
    AD9517_REG_LVPECL_OUT0,
    AD9517_REG_LVPECL_OUT1,
    AD9517_REG_LVPECL_OUT2,
    AD9517_REG_LVPECL_OUT3
};
```
And directly use `lvepcl_out_ch[3]` to access channel 3 for example, so there is no need for arithmetic.

Reply:
Yes, this approach is more readable. Will do.